packages/babel-traverse/README.md

```diff
@@ -1 +1,30 @@
 # babel-traverse
+The Babel Traverse module maintains the overall tree state, and is responsible for replacing, removing, and adding nodes.
+
+Install it by running:
+```
+$ npm install --save babel-traverse
+```
+We can use it alongside Babylon to traverse and update nodes:
+```javascript
+import * as babylon from "babylon";
+import traverse from "babel-traverse";
+
+const code = `function square(n) {
+  return n * n;
+}`;
+
+const ast = babylon.parse(code);
+
+traverse(ast, {
+  enter(path) {
+    if (
+      path.node.type === "Identifier" &&
```

Comment:
FYI you can also do `if (path.isIdentifier({name: "n"})) {` to shorten up this example.